core/transaction/payload/Subscribe.go

```diff
@@ -33,6 +34,9 @@ func (s *Subscribe) Serialize(w io.Writer, version byte) error {
 		serialization.WriteUint32(w, s.Bucket)
 	}
 	serialization.WriteUint32(w, s.Duration)
+	if version == 2 {
```

Comment:
I think we should change version == to version >= especially for all the version == 1